yarn-project/circuit-types/src/logs/l1_note_payload/encrypt_buffer.ts

```diff
@@ -33,21 +35,17 @@ export function deriveAESSecret(ecdhPubKey: PublicKey, ecdhPrivKey: GrumpkinPriv
  * @param data - The data buffer to be encrypted.
  * @param ownerPubKey - The owner's public key as a PublicKey instance.
  * @param ephPrivKey - The ephemeral private key as a Buffer instance.
- * @param curve - The curve instance used for elliptic curve operations.
  * @returns A Buffer containing the encrypted data and the ephemeral public key.
  */
-export function encryptBuffer(
-  data: Buffer,
-  ownerPubKey: PublicKey,
-  ephPrivKey: GrumpkinPrivateKey,
-  curve: Grumpkin,
-): Buffer {
-  const aesSecret = deriveAESSecret(ownerPubKey, ephPrivKey, curve);
+export function encryptBuffer(data: Buffer, ownerPubKey: PublicKey, ephPrivKey: GrumpkinPrivateKey): Buffer {
```

Comment:
The naming of the arguments should better reflect what we also use in the diagram, so lets call it a `ephSk` or the like for the ephemeral secret key.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/6020/commits/e35028bd7e5e07dafa43c6a7b7c532c98ee60e9b